docs/xml-templates.rst

```diff
@@ -171,6 +171,27 @@ Perform multiple tests to render one of several alternatives.  The first matchin
 <div>
 3 is odd</div>
 
+py:match, py:case
+^^^^^^^^^^^^^^^^^
+
+Similar to `py:switch` this makes use of `PEP622 <https://peps.python.org/pep-0622/>`_
+Structural Pattern Matching
+
+>>> import sys, pytest
+>>> if sys.version_info < (3, 10): pytest.skip('pep622 unsupported')
+>>> Template = kajiki.XMLTemplate('''<div>
+... $i is <py:match on="i % 2">
+... <py:case matching="0">even</py:case>
```

Comment:
I might suggest to swap `matching` with `match`, e.g.,

`<py:case match="0">`

Otherwise the syntax is intuitive, nice!